test/functionalTest/cases/000_ngsi10_convenience/individual_context_entity_attributes_put_create.test

```diff
@@ -75,31 +75,29 @@ orionCurl --url "$url" --payload "${payload}" -X "PUT"
 --REGEXPECT--
 +++++ 1. put an entity +++++
 HTTP/1.1 200 OK
-Content-Length: 389
+Content-Length: 203
 Content-Type: application/xml
 Date: REGEX(.*)
 
 <?xml version="1.0"?>
 <updateContextElementResponse>
-  <contextResponseList>
-    <contextAttributeResponse>
-      <contextAttributeList/>
-      <statusCode>
-        <code>404</code>
-        <reasonPhrase>No context element found</reasonPhrase>
-        <details>entity01</details>
-      </statusCode>
-    </contextAttributeResponse>
-  </contextResponseList>
+  <errorCode>
```

Comment:
This was a fix I had to do a few days ago, for the 'without /attribute'. Here it came for free.
Now, if we want it for 'without' we probably want it for 'with' also.